### PR TITLE
Update Protobuf and fix logs

### PR DESCRIPTION
**Pull request by leandroalonso**
Update Protobuf to the latest version to ensure we're using the most up-to-date version.

I also fixed some logs so they report the correct server method used (POST/GET/DELETE) and the URL.

## To test

1. Run the app
2. Login
3. ✅ Perform actions such as subscribing to podcasts and make sure everything is synced

## Checklist

- [x] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [x] I have considered adding unit tests for my changes.
- [x] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.

### CODE REVIEW

File: Modules/Server/Sources/PocketCastsServer/Private/API Tasks/ApiBaseTask.swift [hunk 1]
```diff
@@ -54,7 +54,7 @@ class ApiBaseTask: Operation {
 
             return (responseData, httpResponse.statusCode)
         } catch {
-            FileLog.shared.addMessage("Failed to post to server \(error.localizedDescription)")
+            FileLog.shared.addMessage("Failed to POST to server (\(url) \(error.localizedDescription)")
```
Comment (emilylaguna) [inline Modules/Server/Sources/PocketCastsServer/Private/API Tasks/ApiBaseTask.swift]:
```suggestion
            FileLog.shared.addMessage("Failed to POST to server (\(url)) \(error.localizedDescription)")
```

**Review by emilylaguna — CHANGES_REQUESTED** [review]
Nice, this is working well in my tests of logging in, performing sync actions, etc. 

I have a couple of suggestions:

1. Since we're using the same structure for each of the logs what do you think about moving the logging into a `logFailure` method?
2. What do you think about also including the subclass name in the log? 

Something like: `FileLog.shared.addMessage("[\(type(of: self))] Failed to POST to server (\(url)) \(error)")` which will output: 

`[UpNextSyncTask] Failed to POST to server (https://api.pocketcasts.com/up_next/sync) Error`

I know we can infer which subclass the log came from by looking at the URL but I think including it in the log would help make it explicitly clear.

File: Modules/Server/Sources/PocketCastsServer/Private/API Tasks/ApiBaseTask.swift [hunk 2]
```diff
@@ -88,7 +88,7 @@ class ApiBaseTask: Operation {
 
             return (responseData, httpResponse)
         } catch {
-            FileLog.shared.addMessage("Failed to post to server \(error.localizedDescription)")
+            FileLog.shared.addMessage("Failed to GET from server (\(url) \(error.localizedDescription)")
```
Comment (emilylaguna) [inline Modules/Server/Sources/PocketCastsServer/Private/API Tasks/ApiBaseTask.swift]:
```suggestion
            FileLog.shared.addMessage("Failed to GET from server (\(url)) \(error.localizedDescription)")
```

**Comment by leandroalonso** [comment]
@emilylaguna good suggestion! Done in 112c0bc0ff8c7331bbd2f1dab74e9f9265c7a9c6

**Review by emilylaguna — APPROVED** [review]

**Pull request merged**